content/integrations/jira.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Datadog-Jira Integration
+integration_title: Jira
+kind: integration
+doclevel: basic
+---
+
+### Overview
+
+JIRA is an issue and project tracking system for software teams. This integration allows you to create tickets from triggered alerts in Datadog, and update existing tickets with new information as it arises. Additionally, you can see JIRA ticket creations as events within Datadog to overlay with all of your metrics.
+
+INSERT NEW TICKET EVENT PHOTO HERE
+
+
+
+### Installation
+
+Navigate to your Jira account
+
+Go to settings (Gear icon) –> Applications
+<img src="/static/images/jira/JiraInstallation2.png" style="width: 60%;display:block;"/>
```

Comment:
@technovangelist with max-width the image just looks really really big. Is this ok?